.github/workflows/ci.yaml

```diff
@@ -73,6 +73,14 @@ jobs:
           platforms: linux/amd64, linux/s390x
           file: tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/Dockerfile
 
+  run-nerdctl-tests-on-garm:
```

Comment:
This will probably conflict with #7913, which adds a hunk at the very same line.

Reply:
I've rebased atop of #7913!